lib/custom_widgets/genres.dart

```diff
@@ -1,11 +1,12 @@
 import 'package:flutter/material.dart';
-import '../ui_constants/text_styles.dart';
 
 class WidgetedGenres extends StatelessWidget {
-  final List<String> genres;
+  final List<String?> genres;
```

Comment:
Why the strings could be nullable?

Reply:
because when you search for a value on a map, the return can be the value if the map contains the key associated or otherwise null

Reply:
Then it'd be better to filter out them if null!

Reply:
Done